components/BottomNavigation/examples/BottomNavigationTypicalUseExample.m

```diff
@@ -76,7 +76,9 @@ - (void)commonBottomNavigationTypicalUseExampleInit {
                                       tag:0];
   tabBarItem5.badgeValue = @"999+";
 #if defined(__IPHONE_10_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0)
-  tabBarItem5.badgeColor = [MDCPalette cyanPalette].accent700;
+  if (@available(iOS 10.0, *)) {
```

Comment:
I believe when I added this either Travis or Xcode (compiling for 8.4) complained. Please double-check this works with both.

Reply:
Thanks. My approach doesn't work since @available isn't *available* until the iOS 11 SDK. Switched to a respondsToSelector instead.  Fixes the crash on iOS 8.1 simulators.